README.md

```diff
@@ -1,6 +1,211 @@
-# mongoose-storage-driver-pravega-kvs
-Mongoose driver to test the Pravega Key-Value Store performance
+[![Gitter chat](https://badges.gitter.im/emc-mongoose.png)](https://gitter.im/emc-mongoose)
+[![Issue Tracker](https://img.shields.io/badge/Issue-Tracker-red.svg)](https://mongoose-issues.atlassian.net/jira/software/c/projects/KVS/issues?filter=allissues)
+[![CI status](https://travis-ci.org/emc-mongoose/mongoose-storage-driver-pravega.svg?branch=master)](https://travis-ci.org/emc-mongoose/mongoose-storage-driver-pravega/builds)
+[![Maven metadata URL](https://img.shields.io/maven-metadata/v/http/central.maven.org/maven2/com/github/emc-mongoose/mongoose-storage-driver-pravega/maven-metadata.xml.svg)](http://central.maven.org/maven2/com/github/emc-mongoose/mongoose-storage-driver-pravega-kvs)
+[![Docker Pulls](https://img.shields.io/docker/pulls/emcmongoose/mongoose-storage-driver-pravega.svg)](https://hub.docker.com/r/emcmongoose/mongoose-storage-driver-pravega/)
 
```

Comment:
I see a lot of references to `mongoose-storage-driver-pravega` (CI status, maven metadata, docker pulls). I'd suggest to either change them onto the appropriate `mongoose-srorage-driver-pravega-kvs` links or remove those leaving a "TODO" so it won't confuse anyone.

Reply:
Added a task to see if links get updated when driver is first released